src/ajax/parseXML.js

```diff
@@ -9,7 +9,8 @@ jQuery.parseXML = function( data ) {
 		return null;
 	}
 
-	// Support: IE9
+	// Support: IE 9-11 only
```

Comment:
(I don't care too much about this aspect, though. I just want it settled so that we don't continuously argue about it, changing support comments back & forth)